Internal Field Separator.txt

```diff
@@ -0,0 +1,7 @@
+#Basic IFS example
+#Internal field separator
+#sample program to read words one by one from a txt file using while loop
+#ab.txt contains " Hi I am new to Github"
+while IFS=' ' read key; do
```

Comment:
Best practice is to have a previous IFS and make changes and revert to original IFS